rhscl-postgresql-apb/roles/rhscl-postgresql-apb-openshift/README

```diff
@@ -0,0 +1,41 @@
+Postgresql-Demo-APB-Openshift
```

Comment:
Ya I think we need to congregate our docs in one location so we probably don't need this.  I think a lot of what's in this README can be addressed by a single generic doc.  Maybe we can use the same shell as this doc?